core/templates/pages/learner-dashboard-page/new-learner-dashboard/progress-tab.component.html

```diff
@@ -1,24 +1,104 @@
-<div class="completed-stories" *ngIf="completedStoriesList.length !== 0">
-  <div class="completed-stories-heading">
-    <p class="completed-stories-title" [innerHTML]="'I18N_LEARNER_DASHBOARD_STORIES_COMPLETED_SECTION' | translate">
-    </p>
-    <span class="completed-stories-length">
-      {{completedStoriesList.length}}
-    </span>
-    <img [src]="getStaticImageUrl('/learner_dashboard/star.svg')" alt="star image" class="oppia-svg-star-image">
+<div class="progress-tab">
+  <div class="skill-proficiency" [ngClass]="{'empty-section': emptySkillProficiency}">
+    <div class="skill-proficiency-heading">
+      <p class="skill-proficiency-title" [innerHTML]="'I18N_LEARNER_DASHBOARD_SKILL_PROFICIENCY_SECTION' | translate">
+      </p>
+      <a class="info-tooltip" href="#/">
+        <img [src]="getStaticImageUrl('/learner_dashboard/info.svg')" alt="info image" class="oppia-svg-info-image" *ngIf="!emptySkillProficiency">
+        <span>
+          <div class="badge-info">
+            <img [src]="goldBadgeImageUrl" alt="gold badge image" class="oppia-badge-image">
+            <p class="badge-type">
+              Gold
```

Comment:
Translate this also, ditto below. (unless this is a key for some icon?)

Reply:
Done.